recipe/meta.yaml

```
@@ -41,6 +40,7 @@ requirements:
     - json-c  # [not win]
     - kealib
     - lerc
+    - libarrow
```

... and similarly here (if not staying with `arrow-cpp` generally)
```suggestion
    - libarrow  # [arrow_cpp == "10.0.1"]
```